release-notes.txt

```diff
@@ -1,3 +1,28 @@
+OpenMDAO Version 1.3.0 Alpha Release Notes
+November 4, 2015
+
+API Change
+* The name `jacobian` has been changed everywhere to the clearer, more descriptive,
```

Comment:
really "the function named `jacobian`"    this sounds like you've obliterated the word from the lexicon ;)